projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/metadata/MDType.java

```diff
@@ -83,4 +92,47 @@ public void replace(MDBaseNode oldValue, MDBaseNode newValue) {
             file = newValue;
         }
     }
+
+    // @see: https://github.com/llvm-mirror/llvm/blob/master/include/llvm/BinaryFormat/Dwarf.def
+    public enum DwarfTag {
+        UNKNOWN(-1),
+
+        DW_TAG_ARRAY_TYPE(0x1),
```

Comment:
Just for clarity, could you add comments for which tags correspond to which type class (basic type, derived type or composite type)?